oda_api/api.py

```diff
@@ -179,7 +181,7 @@ def __init__(self,
                  cookies=None,
                  protocol="https",
                  wait=True,
-                 n_max_tries=20,
+                 n_max_tries=200,
```

Comment:
Why is it necessary to have so many re-tries? Don't we risk to have a very long waiting time?

Reply:
We do not simply risk having a very long waiting time, we expect it. 
It better than having user stare in frustration on an in-comprehensive exception!

If it is our fault, it is us who should stare on the exception (in sentry)

We need to choose what should happen in such cases:
* either break and expect user to correct  (retry later? nothing the user can typically do)
* retry and wait until condition resolves itself

Since we can really not expect the user to do anything, waiting (almost indefinitely) is usually the best.
In special cases, like in batch, the behavior might be adjusted, if the jobs should actually break. But most users who prefer not to look deep would prefer to wait.